tests/Components/test_BaseComponent.py

```diff
@@ -23,3 +53,11 @@ def test_append_child(base_component):
     base_component.append_child(child_component)
     assert(base_component.children["wing1"] == child_component)
     assert(base_component.wing1 == child_component)
+
+
+def test_angle_of_attack(freebody_component, attached_component):
```

Comment:
The angle of attack depends both on parent angle, child relative angle and parent velocity. To better test the function, we need to vary these parameters.

You can change them by using `freebody_component.velocity = Vector2(r=12, theta=math.radians(5))`, which will set the parent velocity to 12 m/s in a direction of 5 degrees (polar axis).